apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -175,6 +179,10 @@ func NewFlagSourceConfigurationSpec() (*FlagSourceConfigurationSpec, error) {
 		RolloutOnChange:     nil,
 	}
 
+	// set default value derived from constant default
+	probes := defaultProbesEnabled
+	fsc.ProbesEnabled = &probes
```

Comment:
We could also use the `// +kubebuilder:default` comment in the Spec definition at line 122. 
https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
Adding fields to a previous version feels weird tho

Reply:
Seems this fails as I rely on a pointer to avoid default being set in CRD (defaulting to false if unset, which I want to avoid)

Reply:
I guess we should keep this approach for v1alpha* versions but move away from it when moving to v1beta*

Reply:
Agree :) And refactoring right now is overkill.